scripts/cache_models.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+import os
+import sys
+
+sys.path.insert(0, os.path.dirname(os.path.abspath(os.path.join(__file__, os.pardir))))
+from allennlp.commands import DEFAULT_MODELS
+from allennlp.common.file_utils import cached_path
+
+value = os.environ.get('CACHE_MODELS')
+if isinstance(value, str) and value.lower() == "true":
```

Comment:
it's a bad smell to use `isinstance` when you don't have to. maybe instead

```
cache_models = os.environ.get('CACHE_MODELS', 'false').lower() == 'true'
if cache_models:
  ...
```

Reply:
Good call, thank you!